core/math/vector3.h

```diff
@@ -45,7 +45,7 @@ struct Vector3 {
 
 	union {
 
-#ifdef USE_QUAD_VECTORS
+#if 0
```

Comment:
Such commented out code ends up hard to understand a few months later. It would be nice if you could either add a comment explaining why it's commented out, or simply drop it. Given that it's relatively trivial (just adding one more dimension), I would just drop it, it would be readded if/when this feature is implemented.